packages/website/bin/page-generator/index.js

```diff
@@ -91,9 +101,15 @@ function generatePackagePages(packageInfo, generatorConfig) {
       };
     });
 
+    const env = process.env.NODE_ENV || 'development';
+
     return {
       packageId: pkg.id,
       homePath: path.join('/', homePath),
+      changelogPath:
+        pkg.changelogPath || env === 'development'
+          ? path.join('/', changelogPath)
+          : null,
```

Comment:
Can we not have a binary or operator followed by a ternary? This completely caught me out as to what it actually does?

Follow-up, have we tested this in prod, does it Work As Intended?